ampersand-filtered-subcollection.js

```diff
@@ -20,7 +20,7 @@ var slice = Array.prototype.slice;
 
 function FilteredCollection(collection, spec) {
     this.collection = collection;
-    this.indexes = collection.indexes;
+    this.indexes = collection.indexes || [];
```

Comment:
This `|| []` is done in the `reset` method in collection. idk if we care about that level of consistency.